straxen/analyses/posrec_comparson.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+import pandas as pd
+
+import strax
+import straxen
+
+export, __all__ = strax.exporter()
+
+@straxen.mini_analysis(requires=('event_basics', 'event_posrec_many',))
+def load_corrected_positions(context, run_id, events,
+                             cmt_version='ONLINE',
+                             posrec_algos=['mlp', 'gcn', 'cnn']):
+
+    """
+    Returns the corrected position for each position algorithm available, without the need to reprocess
+    event_basics, as the needed information is already stored in event_posrec_many.
+    
+    :param cmt_version: CMT version to use (it can be a list of same length as posrec_algos, if different versions
+                        are required for different posrec algorithms, default 'ONLINE')
+    :param posrec_algos: list of position reconstruction algorithms to use (default ['mlp', 'gcn', 'cnn'])
+    """
+    
+    posrec_algos = (posrec_algos, ) if isinstance(posrec_algos, str) else posrec_algos
+
+    if hasattr(cmt_version, '__len__') and not isinstance(cmt_version, str) and len(cmt_version) != len(posrec_algos):
+        raise TypeError(f"cmt_version is a list but does not match the posrec_algos ({posrec_algos}) length.")
+        
+    cmt_version = [cmt_version] * len(posrec_algos) if isinstance(cmt_version, str) else cmt_version
+    
+    dtype = []
+    posrec_dependent_dtype = ['x', 'y', 'z', 'r', 'r_naive',
```

Comment:
This variable is not used?

Reply:
I wanted to use it to make the code cleaner when setting results but I ended up not using it. Apparently I forgot it there...